### ISSUE TITLE
unable to download VC 

### ISSUE THREAD

**Issue by Anushree09-N** [mosip/inji#666]
**Describe the bug**
unable to download VC 

**To Reproduce**

1. unlock the app.
2. Click on Add ID
3. Enter VID
4. Enter OTP 
5. Head to HOME screen

**Expected behavior**
VC download should be successful

**Actual behavior**
VC stuck in loading state in the app . Download API returns 404. 

**Screenshots**
![image](https://user-images.githubusercontent.com/79500509/227202717-f1007f6d-89cc-47f0-9cae-f8dc8c0aec25.png)

![image](https://user-images.githubusercontent.com/79500509/227203464-46e45971-8385-4507-8868-e6ab0d2428b1.png)

**Add the screenshot of the profile page with commit id**

![image](https://user-images.githubusercontent.com/79500509/227203525-6f7ac4d5-7990-4125-bc50-e1731aa3e99c.png)

**Smartphone (please complete the following information):**

- Wallet Device: 
Redmi 6A
Android 9
BLE 4.2

- Mimoto version: 1.2.0.1
- MOSIP Version: qa-1201-b2.mosip.net

**Where does the issue occur: Wallet/Verifier?**
wallet

**Logs of wallet and verifier:**
N/A

**Comment by santhoshsunder** [comment]
Tested the new issue now, and the issue is resolved now, we got unblocked, and now we are able to download VC, hence closing the bug, attaching below a screenshot of the latest audit 

screenshot : 
![image](https://user-images.githubusercontent.com/102220709/227423334-2cb55e75-d578-4622-b704-2bff0735e5e8.png)
